src/LiveComponent/doc/index.rst

```diff
@@ -1482,7 +1482,7 @@ this, create your controller like normal, including the submit logic::
 
             return $this->render('post/edit.html.twig', [
                 'post' => $post,
-                'form' => $form,
+                'form' => $form->createView(),
```

Comment:
```suggestion
                'form' => $form, // use $form->createView() in Symfony <6.2
```

Not calling `createView()` in [Symfony 6.2+ is valid](https://symfony.com/blog/new-in-symfony-6-2-dx-improvements#simpler-form-rendering), but in <6.2, it is not.

Reply:
Will LiveComponent support 5.4 when it is marked as non experimental ?

Reply:
> Not calling `createView()` in [Symfony 6.2+ is valid](https://symfony.com/blog/new-in-symfony-6-2-dx-improvements#simpler-form-rendering), but in <6.2, it is not.

Thanks I am indeed using symfony 5.4, i didn't know ->createView() is not needed anymore